cost-analyzer/values.yaml

```diff
@@ -116,6 +116,18 @@ global:
         accumulate: true # entire window resolution
         filters: [] # if no filters, specify empty array
 
+  # Set saved report(s) accessible from reports.html
+  # Ref: http://docs.kubecost.com/saved-reports
+  assetReports:
+    enabled: false # If true, overwrites report parameters set through UI
```

Comment:
On another helm chart PR I opened, I think Webb asked if this were necessary. It doesn't seem like it to me, but the allocation reports and some other things use this pattern and I'm concerned there might be some usecase I'm not imagining.

I guess maybe it's handy to be able to quickly enable / disable things without removing them from your values.yaml?

Reply:
I like these because I think they give you a clear indication that your UI will be overwritten/not used if these are set. But I guess they aren't strictly necessary.